front/app/containers/IdeasEditPage/index.tsx

```diff
@@ -307,7 +311,48 @@ class IdeaEditPage extends PureComponent<Props & InjectedLocalized, State> {
       ] as Promise<any>[]);
 
       clHistory.push(`/ideas/${ideaSlug}`);
-    } catch {}
+    } catch (error) {
+      if (process.env.NODE_ENV === 'development') console.log(error);
+      const apiErrors = error.json.errors;
+      const profanityApiError = apiErrors.base.find(
+        (apiError) => apiError.error === 'includes_banned_words'
+      );
+
+      if (profanityApiError) {
+        const titleProfanityError = profanityApiError.blocked_words.some(
+          (blockedWord) => blockedWord.attribute === 'title_multiloc'
+        );
+        const descriptionProfanityError = profanityApiError.blocked_words.some(
+          (blockedWord) => blockedWord.attribute === 'body_multiloc'
+        );
+
+        if (titleProfanityError) {
+          this.setState({
+            titleProfanityError,
+          });
+        }
+
+        if (descriptionProfanityError) {
+          this.setState({
+            descriptionProfanityError,
+          });
+        }
+      }
+    }
+  };
+
+  onTitleChange = (title: string) => {
+    const { locale } = this.props;
+    const titleMultiloc = { [locale]: title };
```

Comment:
What about other languages that might be part of the multiloc? Wouldn't they get erased here?

Reply:
Agreed